Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector/SaasSecurityEventCollector.yml

```diff
@@ -0,0 +1,69 @@
+name: SaasSecurityEventCollector
```

Comment:
lets rename these ids and files to SaasSecurityModelingRules

Reply:
looks like this is the convention in other integrations

Reply:
done